src/setDragImage.ts

```diff
@@ -1,5 +1,5 @@
 /* eslint-env browser */
-import {default as offset, offsetObject} from './offset' // eslint-disable-line no-unused-vars
+import {default as offset} from './offset' // eslint-disable-line no-unused-vars
```

Comment:
I think the comment `// eslint-disable-line no-unused-vars` can be removed if you remove `offsetObject `